integration-tests/manager-integration-tests/manager-integration-tests-v2/src/test/java/com/redhat/service/smartevents/integration/tests/v2/RunCucumberTest.java

```diff
@@ -13,7 +13,7 @@
 @Suite
 @IncludeEngines("cucumber")
 @SelectClasspathResource("features")
-@ConfigurationParameter(key = Constants.FILTER_TAGS_PROPERTY_NAME, value = "not @ansibleaction")
+@ConfigurationParameter(key = Constants.FILTER_TAGS_PROPERTY_NAME, value = "not @disabled")
```

Comment:
bugfix?

Reply:
I think `@ansibleaction` was a leftover from v1. I changed this annotation and marked as `@disabled` the v2 e2e test